java/apps/authentication-portal/src/main/webapp/privacy_policy.jsp

```diff
@@ -122,6 +122,25 @@
         ToC += "</ul>" + "</nav>";
 
         $("#toc").append(ToC);
+
+        function scrollToTargetElementWithOffset(event) {
```

Comment:
Can't we reuse this function in both places rather than replicating it in both files?

Reply:
https://github.com/wso2/identity-apps/pull/3981/commits/ef806060dc62726a3cd66ac51bc502330eaaafa8